backend/src/main/scala/cromwell/backend/wdl/OutputEvaluator.scala

```diff
@@ -90,7 +91,12 @@ object OutputEvaluator {
       def toError(outputPort: OutputPort) = s"Missing output value for ${outputPort.identifier.fullyQualifiedName.value}"
 
       jobDescriptor.taskCall.expressionBasedOutputPorts.diff(outputs.keySet.toList) match {
-        case Nil => ValidJobOutputs(CallOutputs(outputs))
+        case Nil =>
+          val errorMessagePrefix = "Error applying postMapper in short-circuit output evaluation"
```

Comment:
I'm not sure this will be very helpful to anyone that encounters it but on the other hand it accurately describes what happened 😄

Reply:
Yeah that's certainly true. I wasn't sure what to say here as this code is acting like its generic but it actually exists solely for short-circuit CWL output evaluation yet lives under `cromwell.backend.wdl`. 😱

Reply:
:/ the package name definitely is a pre-wom leftover, if anything it should be `backend/wom`.